quantum/split_common/split_util.c

```diff
@@ -22,30 +26,48 @@
 #    define SPLIT_USB_TIMEOUT_POLL 10
 #endif
 
+#ifdef PROTOCOL_CHIBIOS
+#    define SPLIT_USB_DETECT  // Force this on for now
+#endif
+
 volatile bool isLeftHand = true;
 
-bool waitForUsb(void) {
+#if defined(SPLIT_USB_DETECT)
+#    if defined(PROTOCOL_LUFA)
+static inline bool usbHasActiveConnection(void) { return USB_Device_IsAddressSet(); }
+static inline void usbDisable(void) { USB_Disable(); }
+#    elif defined(PROTOCOL_CHIBIOS)
+static inline bool usbHasActiveConnection(void) { usbGetDriverStateI(&USBD1) == USB_ACTIVE; }
+static inline bool usbDisable() { usbStop(&USBD1) }
+#    else
```

Comment:
```suggestion
#    elif defined(PROTOCOL_VUSB)
static inline bool usbHasActiveConnection(void) { return usbConfiguration; }
static inline void usbDisable(void) { usbDeviceDisconnect(); }
#    else
```
This *might* work.

Reply:
I was going to tackle it as part of another round, to get at least one transport tested. But I figure it compiles, so why not.